installation/ilmt-config.adoc

```diff
@@ -0,0 +1,33 @@
+= Configuring IBM License Metric Tool 
+include::../partials/attributes.adoc[]
+
+
+IBM® License Metric Tool generates reports to help you maintain compliance with your Virtual Processor Core (VPC) subcapacity license terms.
+License Metric Tool can calculate VPC consumption only when software identification tags exist and are activated by the customer.
+The optman global option licenseType must be set to perServer (default value) to activate Virtual Processor Core consumption tracking.
+You can also set the optman global option licenseType to byWorkstation to specify that the licensing type (either perServer or perJob) is specified at creation time for each workstation.
+
+License Metric Tool automatically detects the following IBM Business Automation Manager Open Editions 9.0 release chargeable components that are part of the product:
+
+|=== 
+| Chargeable Software Components
+
+|	Process Automation Manager Open Edition
+|	Decision Manager Open Edition
+|===
+
+To detect and count remote nodes that are managed by IBM Business Automation Manager Open Editions chargeable components, you must manually deploy software tags because no product code is present on those nodes.
```

Comment:
this is not correct, we have the ILMT library today.
which is correct is that the use of the ILMT library has to be added to the project dependencies

Reply:
I'll work on a draft and share here soon

Reply:
Alex, any update here?

Reply:
nope; will come with the codebase changes needed.